lib/scrub/ocn.rb

```diff
@@ -0,0 +1,116 @@
+# frozen_string_literal: true
+
+require "scrub/common"
+require "scrub/max_ocn"
+
+module Scrub
+  class Ocn < Scrub::Common
+    # Take a string, presumed to be one or more OCNs, and get the actual OCNs out as numbers.
+    def initialize(str)
+      @ok_ocns = []
+
+      if str.nil?
+        raise ArgumentError, "ocn is nil"
+      end
+
+      # Get current max oclc
+      @current_max_ocn = Scrub::MaxOcn.new.ocn
```

Comment:
Could we have Scrub::MaxOcn.new raise the error instead? And/or put this in a `def current_max_ocn`

Reply:
Yep, added `Scrub::MaxOcn.current_max_ocn` which does the raising.